Base64 documentation on Benefits Intake V1 (#3599)
department-of-veterans-affairs/vets-api: modules/vba_documents/app/swagger/vba_documents/v1/description.md

        * Note: If, after you've submitted a document, the status hasn't changed to `uploaded` before 15 minutes has elapsed, we recommend retrying the upload in order to make sure the document properly reaches our servers 

 3. Client Request: PUT to the location URL returned in Step 2.

>     * Request body should be encoded as binary multipart/form-data (base64 also available below), equivalent to that generated by an HTML form submission or using “curl -F…”. The format is described in more detail below.

Reviewer: ```suggestion
    * Request body should be encoded as binary multipart/form-data (base64 also available - see details below), equivalent to that generated by an HTML form submission or using “curl -F…”. The format is described in more detail below.
```

The `updated_at` field indicates the last time the status for a given GUID was updated.

### Optional Base64 Encoding

> Base 64 is an encoding scheme that converts binary data into text format so that encoded textual data can be easily transported over network un-corrupted and without any data loss. Base64 is used to take our binary multipart/form-data and encode it in it's entirety in order to bypass certain false positives encountered when going through the VA Firewall. If you feel like you are getting a greater number of PDF files being rejected when uploading them through the PUT request then you might consider encoding the entire payload in base64.

Reviewer: ```suggestion
Base64 is an encoding scheme that converts binary data into text format, so that encoded textual data can be easily transported over network uncorrupted and without data loss. 

Base64 can be used to encode binary multipart/form-data it in its entirety, in order to bypass certain false positives encountered as the VA firewall scans for attack signatures. If your PDF file uploads are failing, causing your GUIDs to expire before a PDF is received, you might consider encoding the entire payload in base64.
```

> After encoding your payload you'll want to preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the type of file you upload so your final string payload would look something like `data:application/pdf;base64,(encryption string)==` making sure that at the end of your payload you close it off with the standard == marker.

Reviewer 2: I wouldnt say they want to add the data:application/pdf;base54. They need to or it wont work

Author: I'll make it required

> After encoding your payload you'll want to preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the type of file you upload so your final string payload would look something like `data:application/pdf;base64,(encryption string)==` making sure that at the end of your payload you close it off with the standard == marker.

Reviewer: ```suggestion
If you use base64 encoding, preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the type of file. The final string payload would look something like `data:application/pdf;base64,(encryption string)==` with the standard == marker to close.
```

Author: @AubreySuter not sure if you meant something different in your suggestion?

The `updated_at` field indicates the last time the status for a given GUID was updated.

### Optional Base64 Encoding

Base64 is an encoding scheme that converts binary data into text format, so that encoded textual data can be easily transported over network uncorrupted and without data loss. 

Base64 can be used to encode binary multipart/form-data it in its entirety, in order to bypass certain false positives encountered as the VA firewall scans for attack signatures. If your PDF file uploads are failing, causing your GUIDs to expire before a PDF is received, you might consider encoding the entire payload in base64.

> After encoding your payload you'll be required to preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the type of file you upload so your final string payload would look something like `data:application/pdf;base64,(encryption string)==` making sure that at the end of your payload you close it off with the standard == marker.

Reviewer: ```suggestion
After encoding your payload, you'll be required to preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the file type. Your final string payload would look something like `data:application/pdf;base64,(encryption string)==` and close with the standard == marker.
```
